checkout-form-manager.js

```diff
@@ -674,7 +662,7 @@ function registerUpsellLinks() {
             return acc;
         }, []);
 
-        urlParams = new URLSearchParams(window.location.search);
+        let urlParams = new URLSearchParams(window.location.search);
```

Comment:
```suggestion
        const urlParams = new URLSearchParams(window.location.search);
```
I've read from a few places that say `const` is preferred as a block-scoped declaration, because it prevents unintended reassignment, but is still mutable (https://evertpot.com/javascript-let-const/)

Reply:
good point. I have updated other places where const could be used in place of var or let. Also made change so we don't need to declare urlParams in every function